src/test/java/com/github/fge/jsonpatch/query/AddQueryOperationTest.java

```diff
@@ -0,0 +1,12 @@
+package com.github.fge.jsonpatch.query;
+
+import com.github.fge.jsonpatch.JsonPatchOperationTest;
+
+import java.io.IOException;
+
+public class AddQueryOperationTest extends Object {
```

Comment:
`extends Object` isn't it by default?

Reply:
This is a bit ugly, because I couldn't ignore a test in testng. It's a special case, because annotation is in a superclass, so two standard methods I used didn't work (or actually worked intermittently).
I added a comment:
`// TODO extend with JsonPatchOperationTest and uncomment constructor when this test needs to be active, couldn't ignore it otherway`

Reply:
I tried @Ignore from testng and @Test(enabled = false)